metadata-ingestion/src/datahub/ingestion/source/affirm/artifact.py

```diff
@@ -20,10 +20,12 @@
 
 
 yaml = YAML(typ='rt')
+logging.basicConfig(stream=sys.stderr, level=logging.INFO)
 
 
 @dataclass
 class AffirmArtifact:
+    schema_name: str
```

Comment:
What is this supposed to be for?

Reply:
Looking later in the PR I get that this is to display the path in the repo for the corresponding artifact. Are we going to do the same for the datasets?

Reply:
hmm `mysql` does not have schema concept so we don't have it.
after re-think it over, I feel it's better not to use `schema` to represent the nested directory structure of `artifacts`. e.g. `logs/application_log.yml`, over here we represent `logs` as database shema-like - and it isn't very much accurate. we can simply name it as `logs.application_log`
wdyt

Reply:
What is the purpose/value for the end-user we plan to get out of this? Can we just provide the file path in the `datahub-metadata` that corresponds to the dataset/artifact in DataHub? That may be useful to cross-compare, update, etc? I suppose it is not possible to have links in properties, but we could also consider adding a link to the GitHub file from elsewhere in DataHub.

Reply:
Btw, this is all going away as we will be moving it to the export-to-datahub script, correct?

Reply:
it wont' go away. it is part of the dataset urn. say `urn:li:dataset:(urn:li:dataPlatform:affirmInfra,development.bluejays,PROD)`.
the difference is either consider the whole `development.bluejays` as the schema.table format, or it as `development` folder, `bluejays` table.